test/gtest/uct/test_uct_perf.cc

```diff
@@ -164,7 +165,8 @@ UCS_TEST_P(test_uct_perf, envelope) {
             test->max *= UCT_PERF_TEST_MULTIPLIER;
             test->min /= UCT_PERF_TEST_MULTIPLIER;
         }
-        run_test(*test, 0, check_perf, GetParam()->tl_name, GetParam()->dev_name);
+        run_test(*test, max_iter, 0, check_perf,
```

Comment:
need to fix existing code+simplify this PR in following way:
1. remove "max_iter" param from run_multi_threaded() in this PR
2. create a copy of "*test" in here, before calling run_test(), and modify its min/max/max_iter fields (to fix pre-existing bug and pass max_iter)

Reply:
thanks! done